BitcoinCore/Classes/BlockHeaders/DifficultyEncoder.swift

```diff
@@ -17,6 +17,29 @@ public class DifficultyEncoder: IDifficultyEncoder {
  */
     public init() {}
 
+    public func compactFrom(hash: Data) -> Int {
```

Comment:
it would be nice to have this change in separate commit with a commit message describing what's changed.

Reply:
Extract to new commit